packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -589,11 +590,19 @@ export class Instance extends Resource implements IInstance {
     this.instancePublicDnsName = this.instance.attrPublicDnsName;
     this.instancePublicIp = this.instance.attrPublicIp;
 
-    if (props.init) {
-      this.applyCloudFormationInit(props.init, props.initOptions);
-    }
+    if (FeatureFlags.of(this).isEnabled(cxapi.EC2_SUM_TIMEOUT_ENABLED)) {
+      this.applyUpdatePolicies(props);
+
+      if (props.init) {
+        this.applyCloudFormationInit(props.init, props.initOptions);
+      }
+    } else {
+      if (props.init) {
+        this.applyCloudFormationInit(props.init, props.initOptions);
+      }
```

Comment:
One more thing before I approve. Can you leave some comments here to explain the sum operation when FF is true and the override, non-breaking behaviour with previous versions when FF is false.

Reply:
Just added!